plugins/woocommerce/src/Internal/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php

```diff
@@ -1153,8 +1153,8 @@ private function add_linked_products_group_blocks() {
 				'attributes' => array(
 					'title'       => __( 'Upsells', 'woocommerce' ),
 					'description' => sprintf(
-						/* translators: %1$s: Learn more about linked products. %2$s: Learn more about linked products.*/
-						__( 'Upsells are typically products that are extra profitable or better quality or more expensive. Experiment with combinations to boost sales. %1$sLearn more about linked products.%2$s', 'woocommerce' ),
+						/* translators: %1$s: Learn more about linked products. %2$s: Learn more about linked products link closing tag.*/
```

Comment:
I like the clarification here... should update `%1$s` as well!

Reply:
Done in the commit [1983981](https://github.com/woocommerce/woocommerce/pull/46346/commits/1983981e9ff6bb1d7440f72a89a918f65e52ad46)